### ISSUE TITLE
Downloading VS Code server locally...  Terminating local server

### ISSUE THREAD

**Issue by eshxcmhk** [microsoft/vscode-remote-release#5605]
First use the remote-ssh, but connection fail. Try to restart VScode and delete the```.vscode-server```directory on the remote server, but the connection is still unsuccessful
```
[19:20:13.640] Log Level: 2
[19:20:13.642] remote-ssh@0.65.8
[19:20:13.642] darwin x64
[19:20:13.643] SSH Resolver called for "ssh-remote+****", attempt 1
[19:20:13.644] "remote.SSH.useLocalServer": true
[19:20:13.644] "remote.SSH.path": undefined
[19:20:13.645] "remote.SSH.configFile": ***/ssh/config
[19:20:13.645] "remote.SSH.useFlock": true
[19:20:13.646] "remote.SSH.lockfilesInTmp": true
[19:20:13.646] "remote.SSH.localServerDownload": always
[19:20:13.646] "remote.SSH.remoteServerListenOnSocket": false
[19:20:13.646] "remote.SSH.showLoginTerminal": true
[19:20:13.647] "remote.SSH.defaultExtensions": []
[19:20:13.647] "remote.SSH.loglevel": 2
[19:20:13.653] SSH Resolver called for host: ****
[19:20:13.653] Setting up SSH remote "****"
[19:20:13.658] Acquiring local install lock: /var/folders/8r/t660sjwd4d1gpmq8r_snrmp00000gn/T/vscode-remote-ssh-2d384c90-install.lock
[19:20:13.663] Looking for existing server data file at /Users/docyp/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-2d384c90-83bd43bc519d15e50************-0.65.8/data.json
[19:20:13.664] Using commit id "83bd43bc519d15e50************" and quality "stable" for server
[19:20:13.671] Install and start server if needed
[19:20:13.678] PATH: /usr/local/opt/rust/cargo/bin:****/flutter/bin:****/android/sdk/ndk/current:****/android/sdk/emulator:****/android/sdk/tools:****/android/sdk/tools/bin:****/android/sdk/platform-tools:/usr/local/opt/libtool/libexec/gnubin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/Library/Apple/usr/bin:/Library/Java/JavaVirtualMachines/jdk1.8.0_291.jdk/Contents/Home/bin:****/gradle/current/bin:****/maven/maven/bin
[19:20:13.678] Checking ssh with "ssh -V"
[19:20:13.693] > OpenSSH_8.1p1, LibreSSL 2.7.3

[19:20:13.695] Using SSH config file "***/ssh/config"
[19:20:13.696] askpass server listening on /var/folders/8r/t660sjwd4d1gpmq8r_snrmp00000gn/T/vscode-ssh-askpass-ac21cae30cdc363b8464bcf2ff6eed7617415859.sock
[19:20:13.696] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/8r/t660sjwd4d1gpmq8r_snrmp00000gn/T/vscode-ssh-askpass-ea0a2aa268580d8a07e17547c01204843f030f2b.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","52810","-o","ConnectTimeout=15","-F","***/ssh/config","****"],"dataFilePath":"/Users/docyp/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-2d384c90-83bd43bc519d15e50************-0.65.8/data.json"}
[19:20:13.697] Local server env: {"DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer)","VSCODE_SSH_ASKPASS_MAIN":"/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/8r/t660sjwd4d1gpmq8r_snrmp00000gn/T/vscode-ssh-askpass-ac21cae30cdc363b8464bcf2ff6eed7617415859.sock"}
[19:20:13.698] Spawned 37072
[19:20:13.910] > local-server-1> Spawned ssh, pid=37082
[19:20:13.915] stderr> OpenSSH_8.1p1, LibreSSL 2.7.3
[19:20:14.083] stderr> debug1: Server host key: ecdsa-sha2-nistp256 SHA256:1VLXweyid2mGqTC8kZI+1cZVQf8ouoIw/8J+c/nyGxs
[19:20:14.356] Got askpass request: {"request":"Password:"}
[19:20:14.357] Showing password prompt
[19:20:14.359] Listening for interwindow password on /var/folders/8r/t660sjwd4d1gpmq8r_snrmp00000gn/T/vscode-ssh-askpass-902fb41cc20d05529baec17cbec6baa41d2e8ca8.sock
[19:20:14.359] Writing password prompt to globalState
[19:20:19.280] Got password response
[19:20:19.280] Interactor gave response: **********
[19:20:19.281] Cleaning up other-window auth server
[19:20:19.355] stderr> Authenticated to ***.***.***.*** ([127.0.0.1]:52811).
[19:20:19.469] > ready: 3d4f0b502233
[19:20:19.486] > Linux 3.10.0-1127.el7.x86_64 #1 SMP Tue Mar 31 23:36:51 UTC 2020
[19:20:19.486] Platform: linux
[19:20:19.535] > 3d4f0b502233: running
[19:20:19.580] > Acquiring lock on /run/user/0/vscode-remote-lock.root.83bd43bc519d15e50************
[19:20:19.595] > Installing to /root/.vscode-server/bin/83bd43bc519d15e50************...
> Trigger local server download
> 3d4f0b502233:trigger_server_download
> platform==linux==
> vscodeArch==x64==
> destFolder==/root/.vscode-server/bin/==
> destFolder2==83bd43bc519d15e50************==
> 3d4f0b502233:trigger_server_download_end
> Waiting for client to transfer server archive...
> Waiting for /root/.vscode-server/bin/83bd43bc519d15e50************/vscode-scp-done.flag and vscode-server.tar.gz to exist
>  
[19:20:19.595] Got request to download on client for {"platform":"linux","arch":"x64","destFolder":"/root/.vscode-server/bin/83bd43bc519d15e50************"}
[19:20:19.596] Downloading VS Code server locally...
[19:20:19.746] Terminating local server
[19:20:19.751] Resolver error: Error: Running the contributed command: '_workbench.downloadResource' failed.
	at _executeContributedCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:111039)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async /Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:427617
	at async Object.t.downloadAndCopyToHost (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:427532)
	at async /Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:434898
	at async Object.e [as tryInstallWithLocalServer] (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:434982)
	at async /Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:452665
	at async Object.t.withShowDetailsEvent (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:456460)
	at async /Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:433816
	at async C (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:430392)
	at async Object.t.resolveWithLocalServer (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:433431)
	at async Object.t.resolve (/Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:454065)
	at async /Users/docyp/.vscode/extensions/ms-vscode-remote.remote-ssh-0.65.8/out/extension.js:1:526942
[19:20:19.754] Local server exit: 15
[19:20:19.759] ------
```

**Comment by eshxcmhk** [comment]
```
Access to XMLHttpRequest at 'https://vscode.cdn.azure.cn/stable/83bd43bc519d15e50c4272c6**********/vscode-server-linux-x64.tar.gz' 
(redirected from 'https://update.code.visualstudio.com/commit%3A83bd43bc519d15e50c4272c6**********/server-linux-x64/stable') from origin 'vscode-file://vscode-app' has been blocked by CORS policy: 
No 'Access-Control-Allow-Origin' header is present on the requested resource.
GET https://vscode.cdn.azure.cn/stable/83bd43bc519d15e50c4272c6**********/vscode-server-linux-x64.tar.gz net::ERR_FAILED
```

**Comment by bamurtaugh** [comment]
Based on `has been blocked by CORS policy`, this may be https://github.com/microsoft/vscode-remote-release/issues/4743, which is now in https://github.com/microsoft/vscode/issues/128492.

I'd recommend commenting on that issue, and I'm going to close this one. If you think there's another issue at play here, please just let me know. Thank you!
